apps/pip/test/base/kustomization.yaml

```diff
@@ -8,6 +8,7 @@ resources:
   - ../../data-management-subscriptions-cron/pip-data-management-subscriptions-cron.yaml
   - ../../account-management-inactive-verification-cron/pip-account-management-inactive-verification-cron.yaml
   - ../../account-management-media-reporting-cron/pip-account-management-media-reporting-cron.yaml
+  - ../../publication-services-mi-reporting-cron/pip-publication-services-mi-reporting-cron.yaml
```

Comment:
You'll also need to update the files in the test/00 and 01 folders, to apply the 00/01 overlays on top of the base yaml above

Reply:
Updated. Thanks for spotting that :)